eZ/Publish/SPI/Tests/FieldType/ImageIntegrationTest.php

```diff
@@ -44,6 +44,9 @@ class ImageIntegrationTest extends FileBaseIntegrationTest
     /** @var \PHPUnit\Framework\MockObject\MockObject */
     private $aliasCleanerMock;
 
+    /** @var \PHPUnit\Framework\MockObject\MockObject */
```

Comment:
```suggestion
    /** @var eZ\Publish\Core\IO\FilePathNormalizer\Flysystem */
```

Unless you configure the mock after the initial creation, which I don't see anywhere.